roles/app/handlers/main.yml

```diff
@@ -1,29 +1,5 @@
-- name: restart rtd_celery_build
-  service: name=rtd_celery_build state=restarted
-  listen: "restart rtd"
-
-- name: restart rtd_celery_docs
-  service: name=rtd_celery_docs state=restarted
-  listen: "restart rtd"
-
-- name: restart rtd_celery_web
-  service: name=rtd_celery_web state=restarted
-  listen: "restart rtd"
-
-- name: restart rtd_gunicorn_web
-  service: name=rtd_gunicorn_web state=restarted
-  listen: "restart rtd"
-
-- name: restart rtd_gunicorn_admin
-  service: name=rtd_gunicorn_admin state=restarted
-  listen: "restart rtd"
-
-- name: restart rtd_gunicorn_api
-  service: name=rtd_gunicorn_api state=restarted
-  listen: "restart rtd"
-
-- name: restart rtd_gunicorn_redirect
-  service: name=rtd_gunicorn_redirect state=restarted
+- name: restart rtd
```

Comment:
Are you losing the ability to restart celery and gunicorn separately?

Reply:
Agree. Fixed by splitting the handler